tower/src/util/rng.rs

```diff
@@ -174,7 +174,7 @@ mod tests {
         let mut r = HasherRng::default();
         match super::sample_floyd2(&mut r, 2) {
             [0, 1] | [1, 0] => (),
-            err => panic!("{err:?}"),
+            err => panic!("{:?}", err),
```

Comment:
Ah, the previous format would be fine if the crate upgraded to 2021 edition -- which it could do with the new MSRV, but that might be a bigger change than desired right now.